### DISCUSSION TITLE
Make custom previews for mediatype

### DISCUSSION THREAD

**Discussion by krebil** [umbraco/Umbraco-CMS#12141] [Features and ideas]
As a developer, I would love to have a way to make a custom preview of files in both the media section and when a file is selected in a media picker.

It would be useful when you have formats that does not have a built-in preview in Umbraco, but where the visual presentation of the media is still essential to understand its function or where the presentation is essential to picking the right item. A good example of this is Lottie files:

Lottie files examples: https://lottiefiles.com/featured
Read more about Lottie: https://airbnb.io/lottie/#/

**Comment by nielslyngsoe** [comment]
Hi @krebil 

Good idea, and I'm happy that we are aligned.

So this is already possible, and its documented right here:
https://apidocs.umbraco.com/v11/ui/#/api/umbraco.services.mediaPreview

Let me know how it goes :-)

**Reply by krebil** [reply]
Seems easy enough, Is there a plan for how this will work in v12?

**Reply by bergmania** [reply]
That should be exactly the same in v12

**Reply by krebil** [reply]
Cool, thanks guys that's very helpful

**Reply by krebil** [reply]
Ah, I meant to ask how it would work in the new backoffice.. which of course is not V12

**Reply by nielslyngsoe** [reply]
So the extension-point will also be available in 'New Backoffice', but the UI implementation will be different. Let me know if that clarified your question. :-)

**Reply by krebil** [reply]
Yeah, that answers everything. Thanks!
